priv/repo/migrations/20210428020716_add_metadata_to_chat_messages.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Glimesh.Repo.Migrations.AddMetadataToChatMessages do
+  use Ecto.Migration
+
+  def change do
+    alter table(:chat_messages) do
+      add :metadata, :map
+    end
```

Comment:
Should this migration all update all of the existing chat messages?

Reply:
Can do, and similar response to https://github.com/Glimesh/glimesh.tv/pull/666#discussion_r631881980